### PR TITLE
ENH: special-case more AD pvs

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: caproto/curio/server.py [hunk 1]
```diff
@@ -512,7 +512,7 @@ def __getitem__(self, pvname):
                 except IOError:
                     logger.exception("Failed to send beacon to %r. Try "
                                      "setting "
-                                     "EPICS_CAS_BEACON_AUTO_ADDR_LIST=no and "
+                                     "EPICS_CAS_AUTO_BEACON_ADDR_LIST=no and "
```
Comment (danielballan) [inline caproto/curio/server.py]: This inconsistency drives me nuts. Thanks for catching it.

File: scripts/spoof-beamline [hunk 2]
```diff
@@ -45,10 +42,22 @@ class ReallyDefaultDict(defaultdict):
...
 
 def fabricate_channel(key):
+    print(f'fabricating {key}', end=' ')
     if 'PluginType' in key:
         for pattern, val in PLUGIN_TYPE_PVS:
             if pattern.search(key):
+                print('as string')
                 return ChannelString(value=val)
+    elif 'ArrayPort' in key:
+        print('as string')
```
Comment (danielballan) [inline scripts/spoof-beamline]: Should these prints go away?

**Comment by tacaswell** [comment]
Prints removed and force-pushed out of history.

**Comment by danielballan** [comment]
There were unusually high number of failures here (7/8) but they are not related to this PR, and no one test failed on more than 3/8 builds:

```
$ python travisfailed.py https://travis-ci.org/NSLS-II/caproto/builds/383176756
383176757 failed BASE=R3.14.12.6 PVA= BUSY=1-6-1 SEQ=2.2.5 ASYN=4-3
383176758 failed BASE=R3.14.12.6 PVA= BUSY=1-6-1 SEQ=2.2.5 ASYN=4-3
383176759 failed BASE=R3.15.5 PVA=4.7.0 BUSY=1-6-1 SEQ=2.2.5 ASYN=4
383176760 failed BASE=R3.15.5 PVA=4.7.0 BUSY=1-6-1 SEQ=2.2.5 ASYN=4
383176762 failed BASE=R3.16.1 PVA=4.7.0 BUSY=1-6-1 SEQ=2.2.5 ASYN=4
383176763 failed BASE=R7.0.1.1 PVA= BUSY=1-7 SEQ=2.2.5 ASYN=4-33 CA
383176764 failed BASE=R7.0.1.1 PVA= BUSY=1-7 SEQ=2.2.5 ASYN=4-33 CA
Failure Count / Tests
---------------------
3 caproto/tests/test_threading_client.py::test_multithreaded_many_subscribe[epics-base-16-iter1]
1 caproto/tests/test_server.py::test_with_caget[numpy-curio-int-ChannelType.STRING]
1 caproto/tests/test_server.py::test_with_caget[numpy-curio-int-ChannelType.GR_CHAR]
1 caproto/tests/test_bench.py::test_many_connections_same_ioc[INFO-trio-connections:{}-100]
1 caproto/tests/test_server.py::test_with_caget[array-curio-int-ChannelType.GR_FLOAT]
1 caproto/tests/test_pyepics_compat.py::testA_CreatedWithConn
1 caproto/tests/test_threading_client.py::test_multithreaded_many_subscribe[epics-base-4-iter2]
```

I think this is just bad luck, but we should keep an eye on it.

**Pull request merged**